system/lib/libcxxabi/src/cxa_personality.cpp

```diff
@@ -798,6 +798,8 @@ static void scan_eh_tab(scan_results &results, _Unwind_Action actions,
                         {
                             // Native exception caught by exception
                             // specification.
+                            assert(actions & _UA_SEARCH_PHASE);
+                            results.ttypeIndex = ttypeIndex;
```

Comment:
This is not related to this PR, but I compared our `cxa_personality.cpp` with llvm 14.0.0 version, this part is missing. I guess it was dropped when we upgraded the library. I don't think Wasm is using this part of the code though, reducing difference from the upstream code seems better.